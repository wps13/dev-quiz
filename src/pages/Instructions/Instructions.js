import React from 'react';
import {SafeAreaView, Text, StyleSheet} from 'react-native';
import {baseStyles} from '../../assets/styles';

const Instructions = () => {
  return (
    <SafeAreaView style={styles.container}>
      <Text style={styles.title}>How to play</Text>
      <Text style={styles.text}>
        To Play this quiz, you should press the 'start' button and select the
        answer that you think that is correct
      </Text>
    </SafeAreaView>
  );
};

export default Instructions;

const styles = StyleSheet.create({
  container: baseStyles.container,
  title: baseStyles.title,
  text: {
    textAlign: 'center',
    margin: 20,
  },
});
