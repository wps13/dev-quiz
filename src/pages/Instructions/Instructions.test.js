import React from 'react';
import {shallow} from 'enzyme';

import Instructions from './Instructions';

describe('Instructions', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Instructions />);
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
