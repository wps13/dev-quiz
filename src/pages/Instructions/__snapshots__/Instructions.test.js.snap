// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions should render correctly 1`] = `
<ForwardRef(SafeAreaView)
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 22,
        "fontWeight": "700",
        "margin": 5,
      }
    }
  >
    How to play
  </Text>
  <Text
    style={
      Object {
        "margin": 20,
        "textAlign": "center",
      }
    }
  >
    To Play this quiz, you should press the 'start' button and select the answer that you think that is correct
  </Text>
</ForwardRef(SafeAreaView)>
`;
